command_line/show_mask_info.py

```diff
@@ -0,0 +1,24 @@
+# LIBTBX_SET_DISPATCHER_NAME dev.dxtbx.show_mask_info
+
+import sys
+
+from dxtbx.model.experiment_list import ExperimentListFactory
+
+
+def show_mask_info():
```

Comment:
Separating this into two separate functions might make the code more reusable elsewhere (but not essential):
1) `def run(args):` method that takes the list of filenames, constructs an experiment list and passes this to the second function
2) `def show_mask_info(experiment_list):` that iterates over the imagesets in the experiment list and prints the mask info.

Reply:
Also 01b904ad3fa9ecf840c2fb5f5b35d8a59e9a038a